nitro-protocol/test/contracts/VirtualPaymentApp/requireStateSupported.test.ts

```diff
@@ -145,7 +145,7 @@ describe('requireStateSupported (candidate plus single proof state route)', () =
     {...vVR, bobAdjustedCorrectly: false, reason: 'Bob not adjusted correctly'},
     {...vVR, nativeAsset: false, reason: 'only native asset allowed'},
     {...vVR, multipleAssets: true, reason: 'only native asset allowed'},
-    {...vVR, aliceUnderflow: true, reason: 'underflow'},
+    {...vVR, aliceUnderflow: true, reason: ' '}, // we expect transaction to revert without a reason string
```

Comment:
Single space here is required in order to trigger the `expectRevert` clause, but doesn't `expectRevert` check the returned error against the given string? Shouldn't it fail here to read the error `" "`?

Reply:
Fair question. The supplied string is used to pattern match (a regex, I think) the error message, which typically comes back as some header plus the string we choose to encode in solidity. Since that header has spaces in it, the pattern matching passes.

Reply:
Somewhat hacky on both ends, but not too tricky to rediscover the logic, and probably not worth breaking out to a different style of check.

:+1: